### PR TITLE
Implement authentication using Windows Credentials Management API

### PR DESCRIPTION
**Pull request by aliozgur**
One more try for #10 and #11.
The actual implementation is  45500ac

### CODE REVIEW

File: src/JenkinsBuildMonitor/AppModel.cs [hunk 1]
```diff
@@ -183,17 +198,62 @@ public void AddServer()
...
+		{
+			SetJobSubscriptions("c");
+		}
+		public void SelectAll()
+		{
+			SetJobSubscriptions("a");
+		}
+
+
+		private void SetJobSubscriptions(string mode)
```
Comment (DavidRogers) [inline src/JenkinsBuildMonitor/AppModel.cs]: `mode` should probably be an enum here.

**Comment by DavidRogers** [comment]
Just so that I am clear about what this code is doing, can you explain what problem this fixes. I have never used the WCM api before.

File: src/JenkinsBuildMonitor/AppModel.cs (hunk 1 above)
Comment (aliozgur) [inline src/JenkinsBuildMonitor/AppModel.cs]: Yes mode can be an enum

**Comment by aliozgur** [comment]
WCM API provides apps an OS level secure vault ( something more or less like Key Chain in Mac OSX) service to store and retreive credentials. Instead of using your own encryption and secure storage methods you can rely on WCM.

**Pull request merged**